editor/project_settings_editor.cpp

```diff
@@ -220,15 +220,24 @@ void ProjectSettingsEditor::_add_feature_overrides() {
 	presets.insert("standalone");
 	presets.insert("32");
 	presets.insert("64");
-	presets.insert("Server"); // Not available as an export platform yet, so it needs to be added manually
+	presets.insert("server"); // Not available as an export platform yet, so it needs to be added manually.
+	presets.insert("linux"); // Not reported in the list of export platforms yet, so it needs to be added manually.
+	presets.insert("bsd"); // Not reported in the list of export platforms yet, so it needs to be added manually.
```

Comment:
I'm not fond of hardcoding those here. They should be registered by the `linuxbsd` platform code if relevant. They're not directly related to this PR though so I'd probably leave them out here and make another PR to assess whether they're wanted and how to include them.

Reply:
I found a different way to address this by modifying `OS::has_feature()` instead. This hack is no longer needed.